src/AspNetCore.Identity.MongoDB/UserClaimsPrincipalFactory.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.Extensions.Options;
+using System;
+using System.Security.Claims;
+using System.Threading.Tasks;
+
+namespace AspNetCore.Identity.MongoDB
+{
+    public class UserClaimsPrincipalFactory<TUser> : IUserClaimsPrincipalFactory<TUser>
```

Comment:
As a side note, this is related to https://github.com/aspnet/Identity/pull/952 (just to keep reference to that issue, no action required on this PR for that).